docs/3.0.0-beta.x/guides/models.md

```diff
@@ -652,13 +652,13 @@ CREATE TABLE `image_morph` (
 ```
 
 - `image_id` is using the name of the first table with the suffix `_id`.
-  - **Attempted value:** It correspond to the id of an `Image` entry.
-- `related_id` is using the attribute name where the relation happens with the suffix `_id`.
-  - **Attempted value:** It correspond to the id of an `Article` or `Product` entry.
-- `related_type` is using the attribute name where the relation happens with the suffix `_type`.
-  - **Attempted value:** It correspond to the table name where the `Article` or `Product` entry is stored.
+  - **Attempted value:** It corresponds to the id of an `Image` entry.
+- `related_id` is using the attribute name where the relation happen with the suffix `_id`.
+  - **Attempted value:** It corresponds to the id of an `Article` or `Product` entry.
```

Comment:
Like "corresponds" here happen should be plural, "happens" as in "...where the relation happens with..."